theme/static/js/hs_resource_table.js

```diff
@@ -382,19 +383,36 @@ function errorLabel(message) {
 
 function label_ajax_submit() {
     var el = $(this);
-    var labelText = $("#txtLabelName").val();
-    var sanitizedLabelText = $("<div/>").html(labelText.trim()).text();
-    if (labelText !== sanitizedLabelText){
-        // red outline and messaging for invalid input
-        $("#txtLabelName").closest(".modal-body").append(errorLabel("This label contains HTML and cannot be saved."))
-        $("#txtLabelName").addClass("invalid-input");
-        return;
+    var formType = el.attr("data-form-type");
+
+    if (formType == "create-label"){
+        var labelText = $("#txtLabelName").val();
+        var sanitizedLabelText = $("<div/>").html(labelText.trim()).text();
+        let message = "";
+        if (labelText.length >= 75){
```

Comment:
Where did the 75 character limit come from?  It makes sense to have a limit but it should also be enforced in the db.  https://github.com/hydroshare/hydroshare/blob/develop/hs_labels/models.py#L92

Reply:
75 was just a limit that made sense to me based on styling (in the tags editor).
Will enforce for db as well thanks

Reply:
Scott believe this commit should help, thanks: [limit labels to 75 char](https://github.com/hydroshare/hydroshare/pull/4534/commits/63d0a39e017fa427e011343a64b62f2b7cfabd60)